Directory.Packages.props

```diff
@@ -76,20 +76,21 @@
     <PackageVersion Include="System.Drawing.Common" Version="8.0.3" />
     <PackageVersion Include="System.IO.Abstractions" Version="17.2.3" />
     <PackageVersion Include="System.IO.Abstractions.TestingHelpers" Version="17.2.3" />
+    <PackageVersion Include="System.IO.Hashing" Version="8.0.0" />
     <PackageVersion Include="System.Management" Version="8.0.0" />
     <PackageVersion Include="System.Management.Automation" Version="7.4.0" />
     <PackageVersion Include="System.Reactive" Version="6.0.0-preview.9" />
     <PackageVersion Include="System.Runtime.Caching" Version="8.0.0" />
-    <!-- Package System.Security.Cryptography.ProtectedData added as a hack for being able to exclude the runtime assets so they don't conflict with 8.0.1. This is a dependency of System.Data.OleDb but the 8.0.1 version wasn't published to nuget. -->
-    <PackageVersion Include="System.Security.Cryptography.ProtectedData" Version="8.0.0" />
+    <PackageVersion Include="System.Security.Cryptography.ProtectedData" Version="8.0.0" /> <!-- Package added as a hack for being able to exclude the runtime assets so they don't conflict with 8.0.1. This is a dependency of System.Data.OleDb but the 8.0.1 version wasn't published to nuget. -->
```

Comment:
@Aaron-Junker , here's an example of a change that I'm pretty sure is a regression. The comments were changed in https://github.com/microsoft/PowerToys/commit/1f743287397980d585e4ff6dee4d0e7f3399daef and now this PR is bringing them back to what they were. I imagine some more might be sprinkled through the PR, that's why I asked to double-check ;) Thank you!